cmd/kvdpa-cli/kvdpa-cli.go

```diff
@@ -86,6 +86,18 @@ func addAction(c *cli.Context) error {
 	return err
 }
 
+func deleteAction(c *cli.Context) error {
+	if c.Args().Len() != 1 {
+		usage := fmt.Sprintf("Usage: %v %v %v \n", c.App.Name, c.Command.Name, c.Command.ArgsUsage)
```

Comment:
Same as for Add command, use auto-generated usage string

Reply:
done